04/index.html

```diff
@@ -2,29 +2,62 @@
 <html lang="pl">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <title>Document</title>
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="global.css">
+    <link rel="stylesheet" media="(max-width:799px)" href="mobile.css">
+    <link rel="stylesheet" media="(min-width:800px)" href="standard.css">
 </head>
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
+
+    <nav class="navigation">
+
+        <label for="menu-switcher" class="hamburger"></label>
+        <input type="checkbox" id="menu-switcher" class="menu__switcher">
+        <ul class="nav">
+            <li class="nav__item">
+                <a href="#">Home</a>
+            </li>
+        
+            <li class="nav__item ">
+                <a href="#" class="arrow">Items</a>
+            
+                <ul class="nav__dropdown dropdown">
+                    <li class="dropdown__item">
+                        <a href="#">Item 1</a>
+                    </li>
+
+                    <li class="dropdown__item arrow__inside">
+                        <a href="#">Item 2</a>
+
+                        <ul class="nav__dropdown--inside dropdown"> 
+                            <li class="dropdown__item">
+                                <a href ="#">Item 2.1</a>
+                            </li>
+                            
+                            <li class="dropdown__item">
+                                <a href ="#">Item 2.2</a>
+                            </li>
+                        </ul>
+
+                    </li>
+                </ul> 
```

Comment:
Raczej na wersji mobilnej zrobiłbym elementy jeden pod drugim (nawet od razu widoczne) ponieważ na małych komórkach może się okazać, że tego miejsca jest za mało aby pokazać np. 3 zagnieżdżenie z prawej strony ;)
Dodam tylko, że skoro jeden pod drugim to można wykorzystać to, że są to elementy blokowe ;)